models/video-models.js

```diff
@@ -1,34 +1,65 @@
 const db = require("../db/connection");
 
-exports.fetchVideos = () => {
-  return db.query("SELECT * FROM videos").then((result) => {
+exports.fetchVideos = (sort_by = "created_at", tags, order) => {
```

Comment:
order doesn't have a default but I remember seeing that you set order=DESC is order isn't given as a query: just checking I've got that right.

Reply:
yeah if order is undefined in the controller function it's set to DESC before being passed to the model